### PR TITLE
Bytyqil nishimwec

### PR DESCRIPTION
**Pull request by Lasicks007**
We fixed syntax errors. added some elif statements, and provided statements if a bad choice is made for the player to die.

### CODE REVIEW

File: t04_refactored.py [hunk 1]
```diff
@@ -111,7 +112,42 @@ def scott_adventure():
 
 def team_1_adv():
     pass
```
Comment (sheggen) [inline t04_refactored.py:114]: Remove this placeholder text

File: t04_refactored.py [hunk 2]
```diff
@@ -111,7 +112,42 @@ def scott_adventure():
...
+    elif Fruit == "Apple":
+        print()
+        print("you've chosen a normal fruit, you live! ")
+        print()
+
+kill_if_dead(dead)
+
+    # TODO Make sure to add the additional check if the user makes the "bad" choice!
+
+    # TODO Don't forget to check if your user is dead at the end of your chapter!
```
Comment (sheggen) [inline t04_refactored.py:150]: Remove these two lines

File: t04_refactored.py (hunk 1 above)
Comment (Lasicks007) [reply t04_refactored.py]: Deleted

File: t04_refactored.py [hunk 3]
```diff
@@ -111,7 +112,42 @@ def scott_adventure():
...
+    elif Fruit == "Orange":
+        print()
+        print("you've chosen the special fruit, you will have fortune and fame! ")
+        print()
+    elif Fruit == "Apple":
+        print()
+        print("you've chosen a normal fruit, you live! ")
+        print()
+
+kill_if_dead(dead)
```
[the remark this answers is not in the document]
Comment (Lasicks007) [reply t04_refactored.py]: Indented

File: t04_refactored.py (hunk 2 above)
Comment (Lasicks007) [reply t04_refactored.py]: Deleted

File: t04_refactored.py [hunk 4]
```diff
@@ -103,15 +103,46 @@ def scott_adventure():
...
         sleep(delay)
 
+
     kill_if_dead(dead)
 
 ###################################################################################
 ###################################################################################
 ###################################################################################
 
 def team_1_adv():
```
[the remark this answers is not in the document]
Comment (sheggen) [reply t04_refactored.py:113]: Docstrings belong here, not at the bottom of the file.

**Pull request closed without merging**